README.md

```diff
@@ -145,6 +147,30 @@ This integer specifies the LUKS key size (in bits).
 ##### `encryption_luks_version`
 This integer specifies the LUKS version to use.
 
+##### `deduplication`
+This specifies whether or not the Virtual Data Optimizer (VDO) will be used.
```

Comment:
We need to make it clear that this option is only implemented for volumes of type `lvm`. Should the blivet module ignore the option where unsupported or fail with an error?

Reply:
Updated documentation so it mentions LVM pool is required for this option.